### PR TITLE
Added wiki and contact info

### PR DESCRIPTION
**Pull request by cfusterbarcelo**
Added some tags to the information regarding the documentation and contact info of deepImageJ to gather it from the BioImage Model Zoo documentation.

### CODE REVIEW

File: manifest.bioimage.io.yaml [hunk 1]
```diff
@@ -29,6 +29,20 @@ config:
...
   url_root: https://raw.githubusercontent.com/deepimagej/models/master
+  docs: https://github.com/deepimagej/deepimagej-plugin/wiki
+  contact:
+    - name: Arrate Muñoz Barrutia
+      email: mamunozb@ing.uc3m.es
+      github: arratemunoz
+      affiliation:
+        - name: UC3M
+					url: https://www.uc3m.es/home
+		- name: Daniel Sage
```
Comment (oeway) [inline manifest.bioimage.io.yaml:40]: It looks like the format is not right. @cfusterbarcelo

Comment (cfusterbarcelo) [reply manifest.bioimage.io.yaml]: Don't know what happened when I copied. I think it's corrected now.

**Review by oeway — APPROVED** [review]
Looks good! Thanks!

**Comment by FynnBe** [comment]
looks like due this PR being from a fork CI didn't run and actually broke validation.
Please run CI prior to merging, see https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks for approving workflows from public forks.

**Pull request merged**